vochain/state.go

```diff
@@ -504,38 +542,150 @@ func (v *State) IsOracle(addr common.Address) (bool, error) {
 
 // setTreasurer saves the Treasurer address to the state
 func (v *State) setTreasurer(address common.Address) error {
```

Comment:
Would be great if you could cherry-pick this commit to reexport SetTreasurer
https://github.com/randomshinichi/vocdoni-node/commit/2b50175fc5458dfb530794a12d373ce4c5387d6d

Reply:
Second this! This would make integration testing with vocone tokens possible